CODEOWNERS

```diff
@@ -14,6 +14,7 @@ src/include/host/*			@ranj063
 src/include/uapi/**			@thesofproject/steering-committee
 src/include/gdb/*			@mrajwa
 src/include/audio/kpb.h			@mrajwa
+src/include/audio/mux.h		@akloniex
 
```

Comment:
alignment seems wrong here. Is it just for me?

Reply:
Looks invalid also in raw view: https://raw.githubusercontent.com/thesofproject/sof/a8f6b2c1ddfafe4e8d0a463cbd25f3e010035b22/CODEOWNERS

Reply:
I had wrong tab size specified in my editor for files without extensions. It's fixed now.